packages/core/src/reporters/json/json-reporter-util.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from 'path';
```

Comment:
This is actually a duplication of the code in `html-reporter-util.ts`. Can we rename that file to `reporter-util` and reuse the `writeFile` implementation from that file?